### DISCUSSION TITLE
differentUserAuthenticated error on login attempt after tokens expiration

### DISCUSSION THREAD

**Discussion by bngrgl** [keycloak/keycloak#19634] [Q&A]
Hello people,

  I have a react web application where I use keycloak-js v. 20.0.5.
  
   I’ve faced with the behaviour which can be reproduced by the following steps:
1. Login into app with credentials of user A.
2. Wait until token and refresh_token get obsolete.
3. Try to access the app and get redirected into login screen.
4. Login with credentials of user B.
5. Receive status 400 and differentUserAuthenticated error

I did setup couple of tests and found out that the only reason to get rid of such behaviour is to remove ‘AUTH_SESSION_ID’ and ‘AUTH_SESSION_ID_LEGACY’.

 I reviewed the interface of keycloak-js, but did not find an option to remove cookies there.
 
  How can I make keycloak client to ignore the existing session cookies and start a new one?
As an alternative, how can I make a Keycloak session to ‘forget’ about attached user?

**Comment by rcrafa** [comment]
I'm having the same problem, did you solve it?
